utility/test/utility.cpp

```diff
@@ -176,4 +177,27 @@ SCENARIO("isHexadecimal") {
     }
 }
 
+SCENARIO("binaryCopy") {
+    // integer representations of floats and doubles have been computed using
+    // http://babbage.cs.qc.cuny.edu/IEEE-754/
+
+    float in1 = 1.23456f;
+    // We assume that a float is coded on 32bits. That assumption is made in
+    // the Parameter Framework.
+    REQUIRE(sizeof(float) == sizeof(uint32_t));
+    auto out1 = utility::binaryCopy<uint32_t>(in1);
+    CHECK(out1 ==  0x3f9e0610);
+
+    // We assume that a double is coded on 64bits. That assumption is made in
+    // the Parameter Framework.
+    uint64_t in2 = 0x408edd3c0cb3420e;
+    REQUIRE(sizeof(double) == sizeof(uint64_t));
+    auto out2 = utility::binaryCopy<double>(in2);
+    CHECK(out2 == 987.65432109876);
```

Comment:
~~~ cpp
CHECK(utility::binaryCopy<double>(0x408edd3c0cb3420e) == 987.65432109876)
~~~
Maybe also test the reverse ?